src/main/scala/org/broadinstitute/dsde/workbench/sam/directory/JndiDirectoryDAO.scala

```diff
@@ -201,6 +202,13 @@ class JndiDirectoryDAO(protected val directoryConfig: DirectoryConfig)(implicit
     groups.toSet
   }
 
+  override def isGroupMember(groupName: SamGroupName, member: SamSubject): Future[Boolean] = withContext { ctx =>
```

Comment:
this is not implemented such that it will handle nested groups. Instead of loading the group and getting the uniqueMembers attribute you need to load the member and pull its Attr.member attribute.